provisioner/network_dec13.yml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
@IPvSean What are all those provisioner files for? Looks like this was an accidental add?

Reply:
yes accidental add, good catch